open-metadata-implementation/access-services/cognos/cognos-server/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <!-- encoders are assigned the type
+             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <logger name="org.odpi.openmetadata.accessservices.cognos" level="INFO" />
```

Comment:
We will presumably get INFO messages in the build - just need to ensure these are kept manageable and don't log any text with ERROR or WARNING to aid in parsing the results.

Further -- note that for the chassis by default all output should be through the audit logging framework - we have logging disabled by default